GruntFile.js

```diff
@@ -132,7 +132,8 @@ module.exports = function(grunt) {
                         'bower_components/d3/d3.js',
                         'src/*.js',
                         'src/models/*.js',
-                        'test/mocha/*.coffee'
+                        'test/mocha/*.coffee',
+	                    'https://cdn.rawgit.com/Kcnarf/d3-beeswarm/fbda9b54/build/d3-beeswarm.min.js',
```

Comment:
Actually no, it's not just for the unit test - it is required for the plot option `observationType` which allows for a "swarm" layout of the observations.

In the [example](https://github.com/novus/nvd3/pull/2070/files#diff-eccc39e392f46ea2e0b3891db31c9b06R7), I manually pull it it - is there a way to automatically include it in the build? 

That question might be obvious, but I'm new to building libraries like this. 😢